benchkv/main.go

```diff
@@ -22,6 +22,8 @@ import (
 	"sync"
 	"time"
 
+	_ "github.com/go-sql-driver/mysql"
```

Comment:
Notice this line, benchkv should have not import "github.com/go-sql-driver/mysql"
but...

when call `make update` , glide vc would remove any unused files in vendor, it doesn't take xxx_test.go files into consideration.

"github.com/go-sql-driver/mysql" is only referenced by server_test.go, so it would be removed.

Then CI would fail because `make test`  can't find this package in vendor.

this is an ugly method to protect "github.com/go-sql-driver/mysql" from been moved from vendor.